project_national/sdr_upgrades_tmy3.yml

```diff
@@ -481,6 +481,38 @@ upgrades:
       - *HP_cooling_efficiency_ducted
       - *100_percent_conditioned_ducts_only
 
+  - upgrade_name: Detailed Performance Data - Typical Cold Climate
+
+    package_apply_logic: *remove_high_rise_shared
+
+    options:
+      - &hvac_heating_efficiency_ashp_typical_cc
+        option: HVAC Heating Efficiency|ASHP, SEER2 17.5, 8.5 HSPF2, Typical Cold Climate
+        apply_logic: &ducted_ASHP_apply_logic_detailed_performance
+          - HVAC Has Ducts|Yes
+
+      - &hvac_cooling_efficiency_ducted_heat_pump_detailed_performance
+        option: HVAC Cooling Efficiency|Ducted Heat Pump
+        apply_logic: *ducted_ASHP_apply_logic_detailed_performance
+      
+      - &hvac_performance_data_ashp
+        option: HVAC Detailed Performance Data|CCHP Ducted
+        apply_logic: *ducted_ASHP_apply_logic_detailed_performance
+
+      - &hvac_heating_efficiency_mshp_typical_cc
+        option: HVAC Heating Efficiency|MSHP, SEER2 22.7, 10.3 HSPF2, Typical Cold Climate
+        apply_logic: &nonducted_MSHP_apply_logic_detailed_performance
+          - HVAC Has Ducts|No
+          - not: HVAC Heating Efficiency|MSHP, SEER 29.3, 14 HSPF
```

Comment:
Why don't any of the other MSHP upgrades have this MSHP called out?

Reply:
For example, why isn't the line `- not: HVAC Heating Efficiency|MSHP, SEER 29.3, 14 HSPF` used in the "ENERGY STAR heat pump with elec backup" upgrade?

Reply:
I borrowed that line from Jon's work, not SDR work. To stay consistent with the SDR yml, I have now removed that line and merged the change into this PR.